### PR TITLE
[android] Changed text for Import bookmarks button to Import bookmarks and tracks

### PR DESCRIPTION
**Pull request by biodranik**
Users can import not only bookmarks but also tracks from GPX or KML files. The current text "Import bookmarks" does not reflect this possibility.

Please check that terminology for bookmarks and tracks is the same in this and other related translations in your language.

### CODE REVIEW

**Review by strump — APPROVED** [review]

**Review by Jean-BaptisteC — APPROVED** [review]
Ok for french translations

File: data/strings/strings.txt [hunk 1]
```diff
@@ -18655,32 +18655,45 @@
...
+    id = Impor penanda dan trek
+    it = Importazione di segnalibri e tracce
+    ja = ブックマークとトラックをインポートする
+    ko = 북마크 및 트랙 가져오기
+    lt = Žymeklių ir takelių importavimas
+    mr = बुकमार्क आणि ट्रॅक आयात करा
+    nb = Importer bokmerker og spor
+    nl = Bladwijzers en tracks importeren
+    pl = Importuj zakładki i utwory
+    pt = Importar marcadores e faixas
```
Comment (matheusgomesms) [inline data/strings/strings.txt:18685]:
```suggestion
    pt = Importar favoritos e trilhas
```

File: data/strings/strings.txt [hunk 2]
```diff
@@ -18655,32 +18655,45 @@
...
-    zh-Hant = 匯入書籤
+    comment = Bookmark categories screen, button that opens folder selection dialog to import KML/KMZ/GPX/KMB files
+    en = Import Bookmarks and Tracks
+    ar = استيراد الإشارات المرجعية والمسارات
+    be = Імпартаваць закладкі і cцежкі
+    bg = Импортиране на отметки и песни
+    ca = Importa marcadors i les pistes
+    cs = Import záložek a stop
+    da = Importer bogmærker og spor
+    de = Lesezeichen und tracks importieren
```
Comment (Mr-Mime) [inline data/strings/strings.txt:18666]:
```suggestion
    de = Lesezeichen und Tracks importieren
```

**Review by Mr-Mime — CHANGES_REQUESTED** [review]
Review for `de`

**Comment by Kazing** [comment]
> Ok for french translations

I feel that `piste` is not the best translation for tracks in this context, `trace` will be better IMO.

The term used for GPX tracks is often traces GPX ou traces GPS

**Comment by biodranik** [comment]
Please do not introduce _new_ terminology. The same word _bookmarks_ and _tracks_ should be used everywhere in a similar context.

File: android/res/values-fr/strings.xml [hunk 3]
```diff
@@ -581,8 +581,8 @@
 	<string name="bookmark_lists_hide_all">Tout masquer</string>
 	<string name="bookmark_lists_show_all">Tout afficher</string>
 	<string name="bookmarks_create_new_group">Créer une nouvelle liste</string>
-	<!-- Bookmark categories screen -->
-	<string name="bookmarks_import">Importer des signets</string>
+	<!-- Bookmark categories screen, button that opens folder selection dialog to import KML/KMZ/GPX/KMB files -->
+	<string name="bookmarks_import">Importer des signets et des pistes</string>
```
Comment (Kazing) [inline android/res/values-fr/strings.xml:585]:
```suggestion
	<string name="bookmarks_import">Importer des signets et des parcours</string>
```

Comment (Kazing) [reply android/res/values-fr/strings.xml:585]: `Parcours` is the translation already used in the UI for tracks

**Review by jimcarst — APPROVED** [review]

File: data/strings/strings.txt (hunk 2 above)
Comment (map-per) [reply data/strings/strings.txt:18666]: tracks are called 'Strecken' in other parts of the UI. This should be kept consistent. 
I'm unsure whether 'Tracks' or 'Strecken' is better. Maybe 'GPS Tracks'?

Comment (Mr-Mime) [reply data/strings/strings.txt:18666]: I agree that it should be kept consistent.
And imho "Strecken" does not sound good.

I would prefer to either use "Tracks" or "Routen", what do you think?!

E.g Komoot calls their tracks "Routen", from what I saw.

Comment (map-per) [reply data/strings/strings.txt:18666]: I don't like "Strecken" as well.
I think "Tracks" is best, as it's the most commonly used word.

Comment (Mr-Mime) [reply data/strings/strings.txt:18666]: I would suggest to stick with `Tracks` here.

A separate PR should be created to change all other occurrences of "Strecke(n)" to "Track(s)", which I can do later today.

**Comment by biodranik** [comment]
Thanks to everyone for your comments and review!

File: data/strings/strings.txt (hunk 2 above)
Comment (Mr-Mime) [reply data/strings/strings.txt:18666]: Created PR: https://github.com/organicmaps/organicmaps/pull/5387

File: android/res/values-es/strings.xml [hunk 4]
```diff
@@ -579,8 +579,8 @@
 	<string name="bookmark_lists_hide_all">Ocultar todo</string>
 	<string name="bookmark_lists_show_all">Mostrar todo</string>
 	<string name="bookmarks_create_new_group">Crear nueva lista</string>
-	<!-- Bookmark categories screen -->
-	<string name="bookmarks_import">Importar marcadores</string>
+	<!-- Bookmark categories screen, button that opens folder selection dialog to import KML/KMZ/GPX/KMB files -->
+	<string name="bookmarks_import">Importar marcadores y pistas</string>
```
Comment (fitojb) [inline android/res/values-es/strings.xml:583]: Apologies for the extremely late comment, but can we change this translation of track to “trazas” (and “traces” in Catalan; singular “traça”) to be consistent with OSM?

**Pull request merged**